source/index.html

```diff
@@ -1,12 +1,212 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Пинк</title>
   </head>
-  <body>
+  <body class="page__body">
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
+    <header class="page-header">
 
+      <nav class="main-nav">
+        <a class="page-header__logo" href="#" >
+          <img class="page-header__logo-image" src="img/header_logo_pink.svg" alt="Логотип" width="74" height="23">
+        </a>
+        <button class="main-nav__toggle"><span class="visually-hidden">Открыть меню</span><img class="page-header__logo-image" src="../source/img/open_menu_button.svg" alt="Логотип" width="50" height="24"></button>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list">
+            <li class="main-nav__list-item main-nav__list-item-current">
+              <a>
+                Главная
+              </a>
+            </li>
+            <li class="main-nav__list-item">
+              <a href="#">
+                Фотографии
+              </a>
+            </li>
+            <li class="main-nav__list-item">
+              <a href="#">
+                Конкурс
+              </a>
+            </li>
+            <li class="main-nav__list-item">
+              <a href="#">
+                Html Academy
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="main-nav__tagline">
+          <p>Взгляните на жизнь иначе!</p>
+            <img src="img/phone_in_hand.jpg" alt="Мир в розовом цвете" width="320" height="286">
+        </div>
+      </nav>
+    </header>
+
+    <main class="page-main">
+
+      <h1 class="visually-hidden">Приложение Pink</h1>
+
+      <div class="page-main__download">
+        <button class="page-main__button-download"><a href="https://www.apple.com/ru/shop">Скачать приложение</a></button>
```

Comment:
```suggestion
        <a href="https://www.apple.com/ru/shop">Скачать приложение</a>
```
Просто ссылка 😏. Внутри кнопки ссылка не может быть.